pages/common/dig.md

```diff
@@ -6,18 +6,26 @@
 
 `dig +short {{hostname.com}}`
 
-- Lookup the mail server associated with a given domain name (MX record):
+- Lookup the mail server(s) associated with a given domain name (MX record):
 
 `dig +short {{hostname.com}} MX`
 
+- Get all types of records for a given domain name:
+
+`dig {{hostname.com}} ANY`
+
 - Specify an alternate DNS server to query (8.8.8.8 is google's public DNS):
 
 `dig @8.8.8.8 {{hostname.com}}`
```

Comment:
While we're at it, shouldn't `8.8.8.8` have `{{` and `}}` around it here too?

Reply:
Sure. Just that 8.8.8.8 was specified in the description. But technically it should be in tokens too.